KILTprotocol/prototype-services
# chore: upgrade dependencies / better health checks

## fixes KILTProtocol/ticket#1082
This PR upgrades nest and adds health checks.

## How to test:
- Try out all endpoints
- Check out `/health` endpoint

## Checklist:
